src/tests/ftest/datamover/dm_dst_create.py

```diff
@@ -31,15 +31,11 @@ def setUp(self):
         self.ior_flags = self.params.get(
             "ior_flags", "/run/ior/*")
         self.test_file = self.ior_cmd.test_file.value
-        self.cont_type = self.params.get(
-            "cont_type", "/run/dcp/*")
-        self.daos_api = self.params.get(
-            "daos_api", "/run/dcp/*")
 
         # For dataset_gen and dataset_verify
         self.obj_list = []
 
-    def run_dm_dst_create(self, tool):
+    def run_dm_dst_create(self, tool, cont_type, api, check_props):
```

Comment:
(pylint-unused-argument) Unused argument 'check_props'

Reply:
All of the verify_cont(... True...)
would need to be verify_cont(... check_props...)